docs/reference/cli.md

```diff
@@ -52,6 +52,79 @@ Available options:
 
 ## Commands
 
+(cli-auth)=
+
+### `auth`
+
+:::{versionadded} 25.09.0-edge
+:::
+
+Manage Seqera Platform authentication.
+
+**Usage**
+
+```console
+$ nextflow auth <subcommand> [options]
+```
+
+**Description**
+
+The `auth` command provides authentication and configuration management for Seqera. It supports OAuth2 authentication for Seqera Cloud and Personal Access Token (PAT) authentication for Seqera Enterprise installations. Credentials are saved to `~/.nextflow/seqera_auth.config`.
+
+**Options**
+
+`-h, -help`
+: Prints the command usage information.
+
+`-u, -url`
+: Specifies your Seqera API endpoint (default: `https://api.cloud.seqera.io`)
+
+**Subcommands**
+
+`login`
+: Authenticates with Seqera, saves credentials, and configures monitoring and workspaces.
```

Comment:
Also the primary compute environment for the workspace, as of last night. But maybe too much detail.